contents/builder-community-incubator.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: PostHog Builder Community Incubator
```

Comment:
Title feels like a bit of a mouthful. Wonder if we just shorten it to: PostHog for Communities (like PostHog for Startups), or just 'PostHog Community Incubator'.